src/controllers/ProductController.java

```diff
@@ -1,15 +1,14 @@
 package controllers;
 
 import models.Product;
+import utils.Rounder;
 import views.SalesView;
 
-// Controller
-public class ProductController {
+public class ProductController extends Rounder {
```

Comment:
I'm not sure that ProductController should extend Rouder. It's not a child for Rounder. Better to keep Rounder separately and expose static method round